content/articles/2017-12-14-styler-1-0-0.Rmd

```diff
@@ -0,0 +1,237 @@
+---
+title: styler 1.0.0
+author: Lorenz Walthert, Kirill Müller
+date: '2017-12-14'
+slug: styler-1.0.0
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/mpdIPhYqZ4Y
+  author: Heng Films
+description: >
+  styler, a package that can format your code according to the tidyverse style guide, is now on CRAN.
+
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE, comment = NA, promt = TRUE)
+knitr::knit_engines$set(list(
+  styler = function(options) {
+    options$comment <- ""
+    knitr::engine_output(
+      options,
+      
+      c("# Before", options$code),
+      c("# After", styler::style_text(options$code))
+    )
+  }
+))
+
+```
+
+
+We're pleased to announce the release of [styler 1.0.0](https://github.com/r-lib/styler/releases/tag/v1.0.0). styler is a source code formatter - a package to format R code according to a style guide. We've implemented the [tidyverse style guide](http://style.tidyverse.org), so you can format your code according to it. A coherent style is important for consistency and legibility. Just as it is important to putspcesbetweenwords. 
+You can install styler from CRAN:
```

Comment:
What do you think of this tweak?

> ... according to a style guide. It defaults to our implementation of the tidyverse style guide, but there is plenty of flexibility for a user to specify their own style. A coherent style ...

Reply:
That's fine for me. I was not sure how much we should emphasise that styler is not a tidyverse-only style guide but if you are open to this, I am happy to update accordingly.

Reply:
I think it's nice to know that tidyverse style is the default, but also that you are not locked in to that.